apps/sms/services/js/moz_mobile_message/messaging_serivce.js

```diff
@@ -0,0 +1,176 @@
+/*global BridgeServiceMixin, Settings */
+'use strict';
+
+if (!('BridgeServiceMixin' in self)) {
+  importScripts('/services/js/bridge_service_mixin.js');
+}
+
+(function(exports) {
+  const SERVICE_NAME = 'messaging-service';
+  const CLIENT_NAME = 'mozMobileMessageShim';
+
+  const METHODS = Object.freeze([
+    'sendSMS', 'sendMMS', 'resendMessage'
```

Comment:
> we need to access it though one more messaging service wrap up because they should be in the same group?

Personally I'd go this way at least for the consistency sake :)